### PR TITLE
EXD-120 - Tag color field as optional

### PR DESCRIPTION
**Pull request by KamilSilk**

### CODE REVIEW

File: src/components/CreateOrModifyTagDialog.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export default function CreateOrModifyTagDialog({
 			description: formValues.description || undefined,
 		};
 
-		currentTagId === 'new'
+		currentTagId === null
```
Comment (MicDeb-Silk) [inline src/components/CreateOrModifyTagDialog.tsx]: Are you sure that the falsy value of currentTagId will only have a null value? What about undefined?

File: src/hooks/useCreateOrModifyTag.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export function useCreateOrModifyTag(
 	const { data: tagDetails } = useQuery({
 		queryKey: ['canvas-tag-details', currentTagId],
 		queryFn: () => ExcaliApi.getTagById(`${currentTagId}`),
-		enabled: Boolean(currentTagId) && currentTagId !== 'new',
+		enabled: Boolean(currentTagId) && currentTagId !== null,
```
Comment (MicDeb-Silk) [inline src/hooks/useCreateOrModifyTag.ts]: This condition does not make sense. Boolean(null) = false

Comment (KamilSilk) [reply src/hooks/useCreateOrModifyTag.ts]: Yeah, you are right. It should be like this 
`currentTagId !== null`

File: src/components/CreateOrModifyTagDialog.tsx (hunk 1 above)
Comment (KamilSilk) [reply src/components/CreateOrModifyTagDialog.tsx]: Yes. But first i have to change pass value condition for currentTagId. First I have to check if the modalProps is not undefined then i pass a string value. When not I pass null

**Pull request merged**